### DISCUSSION TITLE
Colors of content blocks

### DISCUSSION THREAD

**Discussion by fmaussion** [executablebooks/meta#504] [Q&A]
Hi, haven't the content block colors changed in a weird way recently?

Warnings are in yellow (was red), tips in green (was yellow), etc.

https://jupyterbook.org/content/content-blocks.html

**Comment by fmaussion** [comment]
Sorry my post was quickly written, but here is a screenshot:

![image](https://user-images.githubusercontent.com/10050469/135067796-84755201-d2a1-4472-8f9a-bcba58cca4d6.png)

I think "warning" should be red and "tip" should be yellow? If I remember well this happened before, and may be a mismatch issues between the various themes in the epb ecosystem...

**Reply by choldgraf** [reply]
yep - I seem to remember that this was a change in the pydata theme default colors, I agree we should switch back the default values to what they were before. Want to open up an issue about this in the book theme repo?

**Comment by fmaussion** [comment]
Thanks! Here is the issue:
https://github.com/executablebooks/sphinx-book-theme/issues/408
